ktlint-ruleset-experimental/src/test/resources/spec/indent/format-raw-string-trim-indent-expected.kt.spec

```diff
@@ -87,11 +87,10 @@ fun f() {
 }
 
 class C {
-    val CONFIG_COMPACT =
-        """
-        {
-        }
-        """.trimIndent()
+    val CONFIG_COMPACT = """
+    {
+    }
+    """.trimIndent()
```

Comment:
This kinda makes sense and kinda doesn't.
The consistent requirement on the `"""` being lined up would actually be better.